charts/helm-controller-stack/Makefile

```diff
@@ -8,28 +8,44 @@
+.PHONY: generate-helm-controller
+generate-helm-controller: yq kustomize
+	@TMPDIR=$$(mktemp -d) && git clone --branch ${HELM_CONTROLLER_VERSION} --single-branch --depth 1 git@github.com:fluxcd/helm-controller.git $${TMPDIR}/repo && \
 	${YQ} -y '.metadata.name = ("PREFIX-" + .metadata.name)' $${TMPDIR}/repo/config/rbac/role.yaml | \
-		sed "s/PREFIX/{{ include \"helm-controller-stack.fullname\" . }}/" > ./templates/manager_role.yaml && \
-	${KUSTOMIZE} build $${TMPDIR}/repo/config/default/ > $${TMPDIR}/result.yaml && \
-	${YQ} -y 'select(.kind=="CustomResourceDefinition")' $${TMPDIR}/result.yaml > ./crds/crds.yaml && \
-	${YQ} -y 'select(.kind=="ValidatingWebhookConfiguration" or .kind=="MutatingWebhookConfiguration")' $${TMPDIR}/result.yaml | \
+		sed "s/PREFIX/{{ include \"helm-controller-stack.helmControllerFullname\" . }}/" > ./templates/helmController/manager_role.yaml && \
+	${KUSTOMIZE} build $${TMPDIR}/repo/config/default/ > $${TMPDIR}/result_helm_contoller.yaml && \
+	${YQ} -y 'select(.kind=="CustomResourceDefinition")' $${TMPDIR}/result_helm_contoller.yaml > ./crds/crds_helm_contoller.yaml && \
+	${YQ} -y 'select(.kind=="ValidatingWebhookConfiguration" or .kind=="MutatingWebhookConfiguration")' $${TMPDIR}/result_helm_contoller.yaml | \
 		sed -e 's|name: helm-controller-stack|name: {{ include \"helm-controller-stackca-from: {{ .Release.Namespace }}/{{ include \"helm-controller-stack.fullname\" . }}-serving-cert|' \
-			-e 's|namespace: .*|namespace: {{ .Release.Namespace }}|' > ./templates/webhooks.yaml; \
+			-e 's|namespace: .*|namespace: {{ .Release.Namespace }}|' > ./templates/helmController/webhooks.yaml; \
 	rm -rf $${TMPDIR}
 
+.PHONY: generate-source-controller
+generate-source-controller: yq kustomize
+	@TMPDIR=$$(mktemp -d) && git clone --branch ${SOURCE_CONTROLLER_VERSION} --single-branch --depth 1 git@github.com:fluxcd/source-controller.git $${TMPDIR}/repo && \
+	${YQ} -y '.metadata.name = ("PREFIX-" + .metadata.name)' $${TMPDIR}/repo/config/rbac/role.yaml | \
+		sed "s/PREFIX/{{ include \"helm-controller-stack.sourceControllerFullname\" . }}/" > ./templates/sourceController/manager_role.yaml && \
+	${KUSTOMIZE} build $${TMPDIR}/repo/config/default/ > $${TMPDIR}/result_source_contoller.yaml && \
+	${YQ} -y 'select(.kind=="CustomResourceDefinition")' $${TMPDIR}/result_source_contoller.yaml > ./crds/crds_source_contoller.yaml && \
+	${YQ} -y 'select(.kind=="ValidatingWebhookConfiguration" or .kind=="MutatingWebhookConfiguration")' $${TMPDIR}/result_source_contoller.yaml | \
+		sed -e 's|name: helm-controller-stack|name: {{ include \"helm-controller-stackca-from: {{ .Release.Namespace }}/{{ include \"helm-controller-stack.fullname\" . }}-serving-cert|' \
+			-e 's|namespace: .*|namespace: {{ .Release.Namespace }}|' > ./templates/sourceController/webhooks.yaml; \
+	rm -rf $${TMPDIR}
+
+## run generate-source-controller and generate-helm-controller
```

Comment:
This documentation is not very relevant for the user and is not displayed on `make help`